vendor/github.com/avinetworks/sdk/go/session/avisession.go

```diff
@@ -350,8 +482,14 @@ func SetControllerStatusCheckLimits(numRetries, retryInterval int) func(*AviSess
 	}
 }
 
-func SetNoControllerStatusCheck(avisess *AviSession) error {
-	avisess.ctrlStatusCheckDisabled = true
+func DisableControllerStatusCheckOnFailure(controllerStatusCheck bool) func(*AviSession) error {
```

Comment:
Not sure about this but i think we'll have to avoid updating the `avisession.go` file from vendoring. New clients and models can go in. cc: @monotosh-avi

Reply:
The build is failing because of this 
```
17:40:53 # github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils
17:40:53 pkg/utils/avi_rest_utils.go:86:37: undefined: session.SetNoControllerStatusCheck
17:40:53 pkg/utils/avi_rest_utils.go:89:37: undefined: session.SetNoControllerStatusCheck
```

Reply:
Thanks @shubhamavi. Should I instead make a change in avi_rest_utils to use this flag?